js/bitget3.js

```diff
@@ -0,0 +1,2899 @@
+        //                 "price":"150.000000000000",
+        //                 "source":"接口",
+        //                 "state":"canceled",
+        //                 "symbol":"eth_usdt",
+        //                 "type":"buy-limit"
+        //             }
+        //         ]
+        //     }
+        //
+        // swap
+        //
+        //     [
+        //         {
+        //             "symbol":"cmt_ethusdt",
+        //             "size":"1",
+        //             "timestamp":"1595885546770",
+        //             "client_oid":"f3aa81d6-9a4c-4eab-bebe-ebc19da21cf2",
+        //             "createTime":"1595885521200",
+        //             "filled_qty":"0",
+        //             "fee":"0.00000000",
+        //             "order_id":"671758053112020913",
+        //             "price":"150.00",
+        //             "price_avg":"0.00",
+        //             "status":"0",
+        //             "type":"1",
+        //             "order_type":"0",
+        //             "totalProfits":null
+        //         }
+        //     ]
+        //
+        let data = response;
+        if (!Array.isArray (response)) {
+            data = this.safeValue (response, 'data', []);
+        }
+        return this.parseOrders (data, market, undefined, limit);
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        if (code === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchDeposits() requires a currency code argument');
+        }
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'currency': currency['id'],
+            'method': 'deposit_withdraw',
+            'type': 'deposit',
+            'size': 12,
+        };
+        const response = await this.apiGetOrderDepositWithdraw (this.extend (request, params));
+        //
+        //     {
+        //         "status": "ok",
+        //         "data": [
+        //             {
+        //                 "id": 1171,
+        //                 "type": "deposit",
+        //                 "currency": "usdt",
+        //                 "tx_hash": "ed03094b84eafbe4bc16e7ef766ee959885ee5bcb265872baaa9c64e1cf86c2b",
+        //                 "amount": 7.457467,
+        //                 "address": "rae93V8d2mdoUQHwBDBdM4NHCMehRJAsbm",
+        //                 "address_tag": "100040",
+        //                 "fee": 0,
+        //                 "state": "safe",
+        //                 "created_at": 1510912472199,
+        //                 "updated_at": 1511145876575
+        //             },
+        //         ]
+        //     }
+        //
+        const data = this.safeValue (response, 'data', []);
+        return this.parseTransactions (data, currency, since, limit, params);
+    }
+
+    async fetchWithdrawals (code = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
Not working either, same error as `fetchDeposits`

Reply:
Fixed in PR #11907